main.py

```diff
@@ -52,15 +52,24 @@
 credentials = ServiceAccountCredentials.from_json_keyfile_name("pybotnasq_tok.json", scopes)
 
 def get_tickers():
-	API = XTB(user, password)
-	ticker_list = API.get_AllSymbols()
-	ticker_list = [tickerdicts["symbol"].split(".")[0] for tickerdicts in ticker_list["returnData"] if "symbol" in tickerdicts and ".US" in tickerdicts["symbol"] and "CLOSE ONLY" not in tickerdicts["description"] and "(Cboe BZX Real-Time Quote)" in tickerdicts["description"]]
-	ticker_list = sorted(ticker_list)
-	API.logout()
+	"""Function that gathers all tickers available from XTB on us market"""
```

Comment:
Isn't it "US market"?

Reply:
Add details about returned value - function definition and docstring\
PS: Applicable for all functions